tests/zfs-tests/tests/functional/online_offline/online_offline_001_pos.ksh

```diff
@@ -60,13 +61,21 @@ function cleanup
 
 	done
 
-	$KILL $killpid >/dev/null 2>&1
+	if $PS --no-headers -p $killpid > /dev/null; then
+		$KILL $killpid >/dev/null 2>&1
+	fi
```

Comment:
I'm all for solid error handling but was adding this conditional really needed?  Even in the updated code we're always throwing away stderr, stdout, and the exit code.  Why not leave it as it was, the worst case for the existing case is it fails silently and harmlessly.  The worst case for the new code is somehow `ps` fails to list the pid and we don't even try to kill it, which is worse.  Or am I overlooking something?